cypress/integration/instance/InstanceEdit.ts

```diff
@@ -50,5 +50,65 @@ onlyOn(isEnvironmentType(EnvType.Mocked), () => {
             });
         });
     });
+
+    it("Switch from 'Webhook' to 'Ignore' Error handler", () => {
+      //Open 'Instance one' page
+      cy.ouiaId("Instances list table", "PF4/Table")
+        .ouiaId("Instance one", "PF4/TableRow")
+        .within(() => {
+          cy.get(
+            "td:first a[data-testid='tableInstances-linkInstance']"
```

Comment:
@jomarko All tests in this test suite are running with same instance. You can open directly the instance page in `beforeEach`.
See: https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/2c2203e90dbc1a1b633a788ec7afd8700bf60320/cypress/integration/instance/InstancePage.ts#L19

Reply:
thank you for a tip @sjamboro , is a0781ad what you had in mind please?

Reply:
@jomarko yes